README.md

```diff
@@ -1 +1,66 @@
-# sdk-java
\ No newline at end of file
+# CDEvents Java SDK
+
+Java SDK to produce [CDEvents](https://cdevents.dev).
+
+The SDK can be used to create CDEvents and render as CloudEvents to send them to a specific CloudEvents broker
+
+## Add dependency modules
+
+```xml
+<dependency>
+  <groupId>dev.cdevents.sdk-java</groupId>
```

Comment:
Change groupId to `dev.cdevents`

Reply:
Updated to `dev.cdevents`

Reply:
The `groupId` in the doc still points to the old value.

Reply:
Missed to update in the Readme, updated now. Thank you.